packages/db/src/index.ts

```diff
@@ -1,42 +1,10 @@
-import { CamelCasePlugin, Kysely, PostgresDialect } from "kysely"
-import pg, { type PoolConfig } from "pg"
-import type { DB } from "./db.generated"
-export type { DB as Database } from "./db.generated"
-import { getLogger } from "@dotkomonline/logger"
+import { type Prisma, PrismaClient } from "@prisma/client"
+import type { DefaultArgs } from "@prisma/client/runtime/library"
 
-export { createMigrator } from "./migrator"
-const logger = getLogger("@dotkomonline/db")
+export * as dbSchemas from "./generatedSchemas"
```

Comment:
I would prefer if it's just named schemas. I think it's quite unmistakenable what schemas we're talking about here

```suggestion
export * as schemas from "./generatedSchemas"
```